src/pages/List.tsx

```diff
@@ -1,22 +1,22 @@
 import { ListItem } from "../components/ListItem";
 import { Section } from "../components/Section";
-import { useFetchData } from "../hooks/useFetchData";
-import { LocationType } from "../types";
+import { useFetchCollection } from "../hooks/useFetchCollection";
+import { PointType } from "../types";
 
 const ListContent = () => {
-  const { data, isLoading, error } = useFetchData<LocationType>("Locations");
+  const { data, isLoading, error } = useFetchCollection<PointType>("point");
   if (isLoading) {
     return <div>Loading...</div>;
   }
   if (error) {
-    return <div>Chyba: {error}</div>;
+    return <div>Stránka nenalezena: {error}</div>;
   }
   if (!data) {
-    return <div>nemám data</div>;
+    return <div>Tato stránka neobsahuje tento druh dat</div>;
   }
 
-  return data.map((location) => {
-    return <ListItem location={location} key={location.id} />;
+  return data.map((point) => {
```

Comment:
data.map zabalit do fragmentu takze  <> {data.map....}</>

Reply:
a odmazal bych ten druhej return (a slozene zavorky )je to takovy osklivy jak jsou dva za sebou

Reply:
Tohle asi uděláme spolu, jak jsem to odmazala a dala to do fragmentu, tak mi tam naskočilo že point nenalezen.

Reply:
`return <> {data.map((location) => (<ListItem location={location} key={location.id} />) }
</>
`